plugins/knife/knife-box-create.rb

```diff
@@ -97,6 +105,24 @@ def run(supported_boxes = [])
       new_box = box_manager.create_box(role, {index: config[:index], ralf: ((role == "ralf") || config[:ralf]), seagull: config[:seagull]})
       instance_id = new_box.id
 
+      if config[:standalone]
+        # The box is a standalone box, so add it now to the etcd cluster,
+        # and create the shared configuration
+        boxes = find_nodes(chef_environment: config[:environment])
+        if boxes.size == 1
```

Comment:
Regardless of the `--standalone` option, could we just check this value and do the right thing?